authentication/views.py

```
@@ -1,3 +1,293 @@
-from django.shortcuts import render
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.permissions import AllowAny
+from django.contrib.auth import authenticate, get_user_model
+from rest_framework_simplejwt.tokens import RefreshToken
+from drf_spectacular.utils import extend_schema
+from rest_framework import serializers
+from social_django.utils import psa
```

Ruff (0.8.2)

`social_django.utils.psa` imported but unused

Remove unused import: `social_django.utils.psa`

(F401)